docs/extensions.md

```diff
@@ -25,7 +25,7 @@ The [pre-commit framework](https://pre-commit.com/) can run `semgrep` at commit-
 ```yaml
 repos:
 - repo: https://github.com/returntocorp/semgrep
-  rev: 'SEMGREP_VERSION_LATEST'
+  rev: 'v0.91.0'
```

Comment:
Unless the deploy process changed very recently I think you can discard this change?

Reply:
Oh no. Yeah, this was an accident. Fixing.